java-frontend/src/main/java/org/sonar/java/resolve/JavaSymbol.java

```diff
@@ -495,6 +495,36 @@ public String toString() {
   }
 
   /**
+   * Represents a final static field/variable.
+   */
+  public static class FinalStaticVariableJavaSymbol extends VariableJavaSymbol implements FinalStaticVariableSymbol {
+
+    VariableTree declaration;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') "declaration" is the name of a field in "VariableJavaSymbol". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2387)